database/cloud_computing_platforms/oracle.json

```diff
@@ -4,13 +4,13 @@
         "description": "Looking for comprehensive documentation, tutorials, or videos so you can learn about Oracle Cloud Services? You can find all of these resources here in the Oracle Help Center.",
         "url": "https://docs.oracle.com/en/cloud/get-started/index.html",
         "category": "cloud-computing",
-        "subcategory": "oracle"
+        "subcategory": "oracle-cloud"
```

Comment:
It's fine, now the subcategory name matches the title. @Anmol-Baranwal, should we rename the file to `oracle-cloud` from `oracle`? If so, we would need to update the imports as well. What do you think?

It's not required though, but to keep the consistency of the name.

Reply:
That’s be helpful @aftabrehan

Reply:
I guess both is fine as long as category description is there. Feel free to request those changes if you want.

Reply:
@sanjeevkumar321, can you do this please?

Reply:
can i rename the file name like "aws.json" to "aws-cloud.json"  ?

Reply:
@aftabrehan @CBID2 
Let him know :)

Reply:
> can i rename the file name like "aws.json" to "aws-cloud.json" and imports also ?

@sanjeevkumar321, Yes, please.  And other files also. The single rule of thumb is to keep the `filename` the same as the `subcategory` key.

Reply:
Hi @aftabrehan , I already push changes kindly verify it .

Reply:
Looks good.